src/observer.ts

```diff
@@ -60,14 +64,23 @@ export interface DataLayerRule {
   debug?: boolean;
   source: string;
   operators?: OperatorOptions[];
-  destination: string | Function;
+  destination?: string | Function;
   readOnLoad?: boolean;
   url?: string;
   id?: string;
   description?: string;
   monitor?: boolean;
   waitUntil?: number | Function;
   maxRetry?: number;
+  version?: number;
+  fsApi?: string;
```

Comment:
`fsApi` is one of `FS_API_CONSTANTS`, no?  Should that be the type and not `string`.